src/components/blocks/search-form/_search-form.scss

```diff
@@ -8,17 +8,10 @@
     border-radius: 2px;
     padding: 10px 10px 8px;
     position: relative;
-
-    @include mixins.breakpoints-bpMinSmall {
-      padding-top: 10px;
-      padding-bottom: 8px;
-    }
-
-    @include mixins.breakpoints-bpMinMedium {
-      padding-top: 10px;
-      padding-bottom: 16px;
-    }
-
+    margin:auto;
+    width:100%;
+    max-width: 720px;
```

Comment:
very trivial formatting thing here, space after colons, but not worth fixing if it passed linting.

Reply:
Thats really strange - I don't have spaces after these colons in vscode.

Reply:
sorry, i wasn't clear - there are meant to be spaces after colons throughout

Reply:
Ah I see!  I hadn't seen this being used across files but we can defo use that convention going forward.

Reply:
more concerned that this fix for autosuggest will leave the form looking different to where it lands on EtC. this issue should never have gone through design, which always tends to introduce unplanned design changes. this was just something we had overlooked and needed to fix, without design change.